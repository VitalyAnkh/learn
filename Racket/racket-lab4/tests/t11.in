(set? '())
(set? '(x))
(set? '(x y z))
(set? '(x x x))
(set? '(x y z x))
