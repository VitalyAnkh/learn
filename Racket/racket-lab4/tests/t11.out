#t
#t
#t
#f
#f
